src/app/components/UserAssets/index.tsx

```diff
@@ -254,8 +244,8 @@ function AssetRow({
       </td>
       <td className="tw-text-right tw-hidden md:tw-table-cell">
         <LoadableValue
-          value={numberToUSD(Number(weiTo4(dollarValue)), 4)}
-          loading={dollars.loading}
+          value={numberToUSD(Number(weiTo4(dollarValue.value)), 4)}
```

Comment:
Everything looks good now but I found all of the references of `numberToUSD` and I found out all of them were used with 4 decimals. Could you adjust `numberToUSD` function to have 4 as a default decimal value and then you can use it without the parameter? Also, please adjust all of the places where it's used so we don't have any new occurrences of it.

Reply:
Not sure all of numberToUSD are using 4, as I see in the homepage there is 2, 3 or even 0 decimal places used

Reply:
I can see this when I find all the occurrences but I think it's worth the refactoring even if we have some places where it would be different (that's why we have it parametrized) :)

![image](https://user-images.githubusercontent.com/14878713/133979143-cce7228d-d581-4261-a172-8394a716f69d.png)

Reply:
Ok, refactored and made new helper `weiToUSD` so there wouldn't be need to use numberToUSD, weiToFixed and Number each time.